tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/core/utils.h

```diff
@@ -426,12 +426,18 @@ inline CompositeInstruction generateSeed(const CompositeInstruction& instruction
  * @param flattened Vector of instructions representing the full flattened composite
  * @param composite Composite instruction to be flattened
  */
-inline void FlattenHelper(std::vector<std::reference_wrapper<Instruction>>& flattened, CompositeInstruction& composite)
+inline void FlattenHelper(std::vector<std::reference_wrapper<Instruction>>& flattened,
```

Comment:
Should this be moved to the command_language_utils.h/cpp?

Reply:
Probably. I'll rebase and move it.